### ISSUE TITLE
Tab context menu assumes it's used on the active/selected tab

### ISSUE THREAD

**Issue by mnoorenberghe** [totallymike/contextPlus#25]
If I right-click on a background/inactive tab the menu items assume I'm right-clicking on the selected/active tab which is very wrong.

I'm not sure if the web extension API allows you to be called when the context menu is opening but that would be useful. In the meantime I suggest including all contexts in the list, not excluding the ones for the selected tab.

**Comment by totallymike** [comment]
Hm...that's a good point.  Think we share a context menu that's recreated every time one changes tabs. We may have to rethink sharing page and tab context.

**Comment by duttyend** [comment]
I was going to open this issue, thanks for doing it and thank you @totallymike by advance !

**Comment by gabrielfin** [comment]
I don't think this is possible until this feature is implemented: https://bugzilla.mozilla.org/show_bug.cgi?id=1215376

**Comment by jryans** [comment]
It appears [bug 1215376](https://bugzilla.mozilla.org/show_bug.cgi?id=1215376) has recently been fixed in Firefox 60, so it may be possible to fix this issue now.
